tests/output/ConfWriterTest.py

```diff
@@ -25,7 +25,19 @@ class ConfWriterTest(unittest.TestCase):
                     '    key\\\\backslash = value\\\\backslash\n'
                     '    key\\,comma = value,comma\n'
                     '    key\\#hash = value\\#hash\n'
-                    '    key\\.dot = value.dot\n')
+                    '    key\\.dot = value.dot\n'
+                    '    a_default = val, val2\n')
+
+    example_file_2 = ('[defaults]\n'
```

Comment:
can we not enumerate them but give them proper names? append_example_file or so?